.github/scripts/generate_ci_workflows.py

```diff
@@ -445,6 +447,14 @@ def generate_workflow_file(self, workflow_template: jinja2.Template) -> None:
         (jinja_env.get_template("windows_ci_workflow.yml.j2"), WINDOWS_WORKFLOWS),
         (jinja_env.get_template("bazel_ci_workflow.yml.j2"), BAZEL_WORKFLOWS),
     ]
+    # Delete the existing generated files first, this should align with .gitattributes file description.
+    existing_workflows = glob.glob(str(GITHUB_DIR / "workflows/generated-*"))
```

Comment:
nit: `Path.glob()` is a thing
```suggestion
    existing_workflows = GITHUB_DIR.glob("workflows/generated-*")
```